plugins/modules/sefcontext.py

```diff
@@ -39,21 +41,28 @@
     default: a
   setype:
     description:
-    - SELinux type for the specified target.
+    - SELinux type for the specified I(target).
+    type: str
+  equal:
+    description:
+    - Path to use for substituting the I(target). The context labeling for the I(target) subtree is made equivalent to this path.
+    version_added: 6.3.0
     type: str
-    required: true
   seuser:
     description:
-    - SELinux user for the specified target.
+    - SELinux user for the specified I(target).
+    - Defaults to C(system_u) for new file contexts and to existing value when modifying file contexts.
     type: str
   selevel:
     description:
-    - SELinux range for the specified target.
+    - SELinux range for the specified I(target).
+    - Defaults to C(s0) for new file contexts and to existing value when modifying file contexts.
     type: str
     aliases: [ serange ]
   state:
     description:
     - Whether the SELinux file context must be C(absent) or C(present).
+    - Specifying C(absent) deletes labeling mappings to both SELinux types and path substitutions.
```

Comment:
Not sure exactly how to refer it, do you mean to modify it to work with `setype` arg present the same way as deletion with `equal` present? E.g.:
```
    - Specifying C(absent) without either I(setype) or I(equal) deletes both SELinux type or path substitution mappings that match I(target).
```

And in code something like
```
if exists and ((equal is None and setype is None) or exists[2] == setype):
```
(since with `equal` together with `state=absent` already works like this)

Reply:
Now read my comment again assuming I am a schmuck who knows nothing of SELinux on the other side. Well, that is actually the reality. :-)

I was thinking only of the docs, if to try and make any sense out of my comment you are going to the code, I think my comment is probably way off the mark. Feel free to ignore it.

Reply:
Well this definitely needs some updating since I modified how it works together with `equal`. I was always thinking that it might be better to modify deletion to work the same way together with `setype`.

What do you think of the updated documentation text in https://github.com/ansible-collections/community.general/pull/5830#discussion_r1071042051 compared to the original you commented on?

Reply:
Well that was only because I did not understand what did you mean exactly with "refer explicitly". However, I've changed the deletion logic to be the same for both 2 types of mappings and used the documentation text I proposed earlier.

Documentation should mean the source code don't need to be looked at. Is the now changed doc text more clear?